### ISSUE TITLE
When impersonating another user, display current user as well as original login

### ISSUE THREAD

**Issue by takkaria** [emoncms/group#42]
At the moment when you impersonate another user, it says "You were originally logged in as *username*."  It would be useful if it said "Currently logged in as *username*" as well.

**Comment by pb66** [comment]
I noticed this too.

I also noticed that the current warning in the navbar takes up a huge amount of space that was already tight if you have several dashboards to link to.

I would like to see the far right "login/log out" button re-purposed to show who is logged like most online accounts seem to, and when clicked it drops down to show links to account page, groups? and then a "log out" option at the bottom.

Perhaps the user name/gravitar background could turn red if logged in as someone else?

I also think the current account "name" should always be displayed (I have it on the far left of the nav bar) so that when you have several accounts open to view (public pages or using apikeys) you always know what account you are looking at, it also doubles as a home button to take you back to the "main" dashboard.
